mimesis/builtins/ru.py

```diff
@@ -150,5 +150,9 @@ def bic(self) -> str:
         :Example:
             044025575.
         """
-        mask = '04#######'
+        country_code = '04'
+        сode = str(self.random.randint(01, 99))
+        bank_number = str(self.random.randint(00, 99))
+        bank_office = str(self.random.randint(050, 999))
```

Comment:
Invalid syntax. We should use `format`.

Reply:
thx
close this PR and create new then all test pass

Reply:
@axce1 Okay, sure!